### ISSUE TITLE
[FEATURE] Variables to refactor

### ISSUE THREAD

**Issue by andrespias** [UdelaRInterior/ansible-backuppc-client#1]
The names of the role variables could be confused with the backuppc_server role. Also, is missing the prefix backuppc_ in backup_state and include_files

**Comment by ulvida** [comment]
Indeed. However, the refactor of variables will lead to a change of the major version of the role. 

Moreover the names, let's also analyse the whole role to see if it can be improved.

Particularly, we should take into consideration [the problem raised in PR](https://github.com/UdelaRInterior/ansible-backuppc-client/pull/3#issue-372549673) when addressing the issue #2.

**Comment by ulvida** [comment]
> Also, is missing the prefix backuppc_ in backup_state and include_files

This is addressed in [v2.0.0](https://github.com/UdelaRInterior/ansible-backuppc-client/releases/tag/v2.0.0). 
However, I missed the problem of possible variables duplication with role `backuppc_server`. We should address this when refactoring this role. I put [an issue](https://github.com/UdelaRInterior/ansible-backuppc/issues/3).
